README.md

```diff
@@ -27,6 +37,18 @@ This repository contains the Dockerfiles I use for Akeneo and other Symfony deve
 
 - **akeneo/akeneo** and **akeneo/akeneo-behat** have been replaced by **akeneo/akeneo-apache**.
 
+### Where to get help:
+
+The Akeneo Community [forum](https://www.akeneo.com/forums/), the Akeneo Community [Slack](https://akeneopim-ug.herokuapp.com)
+
+### Where to file issues:
```

Comment:
Careful, those two subsections duplicates the one line 43 (65 in your PR branch) "How to use these images?". You should merge both.

Maybe move the "How to use these images?" section here, before "GitHub branches and corresponding Docker Hub tags".